libs/electroniccats-escornabot-makech/pxt.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "electroniccats-escornabot-makech",
+    "description": "Electronic Cats Escornabot Makech",
+    "files": [
+        "README.md",
+        "device.d.ts",
+        "config.ts",
+        "ns.ts",
+        "board.json",
+        "board.svg"
+    ],
+    "dependencies": {
+        "core---samd": "file:../core---samd",
+        "buttons": "file:../buttons",
+        "mixer---samd": "file:../mixer---samd"
```

Comment:
does it come with the ESP32 built-in? then you should add a reference here too

Reply:
ESP32 is optional